controllers/AuthController.js

```diff
@@ -20,32 +20,31 @@ class AuthController {
    * @returns {Object} Returns the response
    */
   static async signup(req, res) {
-    let user;
+    let userModel;
     let token;
-    const { body } = req;
+    const {
+      body: { user }
+    } = req;
     try {
-      user = await User.findOne({
-        where: { [Op.or]: [{ email: body.email }, { username: body.username }] }
+      userModel = await User.findOne({
+        where: { [Op.or]: [{ email: user.email }, { username: user.username }] }
       });
-      if (user) {
-        return res.status(401).json({ status: 401, message: 'Account already exist' });
+      if (userModel) {
+        return res.status(401).json({ message: 'Account already exist' });
       }
-      const password = await bcrypt.hash(body.password, 10);
+      const password = await bcrypt.hash(user.password, 10);
 
-      user = await User.create({ ...body, password });
+      userModel = await User.create({ ...user, password });
 
-      token = jwt.sign({ id: user.id, userType: user.userType }, JWT_SECRET);
+      token = jwt.sign({ id: userModel.get().id, userType: userModel.get().userType }, JWT_SECRET);
     } catch (error) {
-      return res.status(401).json({ status: 401, message: 'Please try again' });
+      return res.status(401).json({ message: 'Please try again' });
     }
 
     res.cookie('jwt', jwt, { httpOnly: true, secure: true });
-    const { password, ...userData } = user.get();
+    const { password, ...userData } = userModel.get();
     return res.status(201).json({
-      status: 201,
-      message: 'Account created sucessfully',
```

Comment:
The message was removed because it doesn't appear in ReadMe signup response body.

Reply:
Thank you for noting that @mcaleb808 .

So please remember that we are using Alphas examples on the README as examples, to guide us on how we are meant to format our work. But if we find that adding a status code and message is important in a particular response then we do so.

In this response, a status code, as well as message, are necessary. So please add them back.